### ISSUE TITLE
App Proxy - Header Code 302

### ISSUE THREAD

**Issue by ihateonions-uk** [osiset/laravel-shopify#621]
First off, thanks for this framework and your time invested in maintaining it. Its much appreciated!

## Expected Behavior

To return a response back to Shopify with a 200 header status and not 302 when using an app proxy.

## Current Behaviour

Disclaimer, ive used your example from the usage page for app proxies and still won't work for me.

--

When I run the page on the Shopify store that triggers the app proxy (little bit of Ajax), Im returned a header status code of 302. I believe this is something todo with the auth.proxy maybe? And something going wrong in the redirecting maybe?

The routes and controllers all work fine, and im returning a 200 response as advised but it still errors with 302. If I visit the url sent from Shopify (my Ajax to the proxy route) in a new tab it works fine, but from within Shopify it doesn't work. Any thoughts?

Is it something todo with the Header Response 'location' below which is the authentication? And this location url actually returns a 200 status so I know that bit is working fine, but I wonder why it then doesn't continue or complete the proxy request.

**The general header response is,**

`Request URL: https://example.wehateonions.com/proxy?type=subscriber_request&email=test@test.com&product_id=4642359935056&shop=myshop.myshopify.com&path_prefix=%2Fapps%2Fbisa_subscriber&timestamp=1606072483&signature=909510106745cbcbe1f0a4bb35d567bab212ecfbacd11b44895ab739de0b77f1

Request Method: GET

Status Code: 302 

Remote Address: 188.xx.xx.156:443

Referrer Policy: strict-origin-when-cross-origin

`

**The header responses are,**

`access-control-allow-origin: *

cache-control: private, must-revalidate

content-type: text/html; charset=UTF-8

date: Sun, 22 Nov 2020 19:27:19 GMT

expires: -1

location: https://example.wehateonions.com/authenticate/oauth?shop=myshop.myshopify.com

pragma: no-cache

server: nginx`

### Steps to Reproduce

Basically the exact process you have on the usage wiki for proxies, and even with your response example it still won't work for me within shopify. https://github.com/osiset/laravel-shopify/wiki/App-Proxies

Im using AJAX GET to send request to my /proxy route.

### Context

Running this on NGINX live remote server.
Ive enabled Cors Headers so it allows the request which works fine.

* Package Version: v14.0
* Laravel Version: v6.3.0
* PHP Version: v7.4

**Comment by ihateonions-uk** [comment]
For anyone else who gets this problem, it was because I had the proxy route within the 'Route::middleware(['auth.shopify'])->group' and when using the auth.proxy middleware it should be outside of this route group.
